packages/documentation/src/stories/elements/paragraph/paragraph.stories.ts

```diff
@@ -0,0 +1,34 @@
+import type { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { MetaExtended } from '@root/types';
+
+const meta: MetaExtended = {
+  id: '7ecd87f1-de96-4e39-a057-ba1798eb69593',
+  title: 'Foundations/Typography/Paragraph',
+  parameters: {
+    badges: [],
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=26-49',
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const Default: Story = {
```

Comment:
Wrap your story in a div (which does end up in the DOM, but not in the code panel), this will allow you to show the `margin-block-start` reset for `p:first-child`:

```suggestion
export const Default: Story = {
  decorators: [
    (story: StoryFn, context: StoryContext) => html` <div>${story(context.args, context)}</div> `
  ],
```

Reply:
Don't forget to import `StoryFn` and `StoryContext`.